doc/safegcd_implementation.md

```diff
@@ -576,7 +576,7 @@ i = N # divsteps left to do
 while True:
     # Get rid of all bottom zeros at once. In the first iteration, g may be odd and the following
     # lines have no effect (until "if eta < 0").
-    zeros = min(i, count_trailing_zeros(g))
+    zeros = i if g == 0 else min(i, count_trailing_zeros(g))
```

Comment:
Hmm, it feels like this is mostly due to `count_trailing_zeros` giving the unintuitive *-1* as result when fed *0*.

Maybe we should have `count_trailing_zeros` return *N* on *0*?

Reply:
Yeah, that's much more clear!